content/v2.0/monitor-alert/templates/AWS.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Monitor Amazon Web Services (AWS)
+description: >
+  Use the AWS CloudWatch Monitoring template to monitor data from Amazon Web Services (AWS), Amazon Elastic Compute Cloud (EC2), and Amazon Elastic Load Balancing (ELB) with the AWS CloudWatch Service.
+menu:
+  v2_0:
+    parent: Monitor with templates
+    name: AWS CloudWatch
+weight: 201
+---
+
+Use the [AWS CloudWatch Monitoring template](https://github.com/influxdata/community-templates/tree/master/aws_cloudwatch) to monitor data from [Amazon Web Services (AWS)](https://aws.amazon.com/), [Amazon Elastic Compute Cloud (EC2)](https://aws.amazon.com/ec2/), and [Amazon Elastic Load Balancing (ELB)](https://aws.amazon.com/elasticloadbalancing/) with the [AWS CloudWatch Service](https://aws.amazon.com/cloudwatch/).
+
+The AWS CloudWatch Monitoring template includes the following:
+
+- 2 dashboards:
+  - **AWS CloudWatch NLB (Network Load Balancers) Monitoring**: Displays data from the `cloudwatch_aws_network_elb measurement`
+  - **AWS CloudWatch Instance Monitoring**: Displays data from the `cloudwatch_aws_ec2` measurement
+- 2 labels: `inputs.cloudwatch`, `AWS`
+- 1 variable: `v.bucket`
+- 1 Telegraf input plugin: [AWS CloudWatch](/v2.0/reference/telegraf-plugins/#cloudwatch)
+
```

Comment:
Do we want to write these numbers out as they're under 10?

Reply:
Based on the [Google style guide](https://developers.google.com/style/numbers#numbers-as-numerals), I think these fall under the category of "technical quantities" and should be left as-is.

Reply:
yes, thank you @noramullen1!